bat-utils/lib/runtime-wallet.js

```diff
@@ -514,8 +514,15 @@ Wallet.providers.simplex = {
       if (!rate) return
 
       amount = Math.ceil(this.currency.alt2fiat(currency, amount, fiat, true))
-      currency = fiat
+    } else if (currency !== fiat) {
+      rate = underscore.pick(this.currency.fxrates[currency])
+      if (!rate) return
+
+      amount = Math.ceil(amount / rate)
     }
+    currency = fiat
+    min = this.runtime.config.simplex['MIN_' + fiat.toUpperCase()] || 5
```

Comment:
where do we set this?

Reply:
this can be const

Reply:
by `this` do you mean `this.runtime...`? the answer is that it is set during server initialzation. look at `config.js`

i have made `min` a constant... however, my coding style is to put `const` at the beginning of a block, so you'll notice the assignment is a bit more noisy.

Reply:
I mean that I don't see MIN_USD in config.js

Reply:
right. the default is 5. if we want to override that, we can add something to config.js

Reply:
ok so this is just for future use, got it